src/main/resources/application.yaml

```diff
@@ -221,6 +223,9 @@ security:
       - "submitClarifyingQuestionAnswers"
       - "editTimeExtension"
       - "submitTimeExtension"
+      - "awaitingCmaRequirements"
```

Comment:
Think you may have finished for the day before I asked this on slack but what is the awaitingCmaRequirements event. We don't normally have anything both the citizen and TCW can do so don't think this is correct being here and above.